pymc/pytensorf.py

```diff
@@ -1046,14 +1047,26 @@ def find_default_update(clients, rng: Variable) -> Union[None, Variable]:
                 raise ValueError(
                     f"No update mapping found for RNG used in SymbolicRandomVariable Op {client.op}"
                 )
+        elif isinstance(client.op, Scan):
```

Comment:
Futureproofing question: does the new scalar scan from pymc-devs/pytensor#283 subclass from `Scan` (so it will be caught here?)

Reply:
No it doesn't, but ScalarScan does now allows RNGs in the inner graph